src/main/java/github/scarsz/discordsrv/objects/threads/ChannelUpdater.java

```diff
@@ -0,0 +1,157 @@
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * END
+ */
+
+package github.scarsz.discordsrv.objects.threads;
+
+import github.scarsz.discordsrv.Debug;
+import github.scarsz.discordsrv.DiscordSRV;
+import github.scarsz.discordsrv.util.DiscordUtil;
+import github.scarsz.discordsrv.util.PlaceholderUtil;
+import lombok.Getter;
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.entities.GuildChannel;
+import net.dv8tion.jda.api.exceptions.PermissionException;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+public class ChannelUpdater extends Thread {
+
+    @Getter private final Set<UpdaterChannel> updaterChannels = new HashSet<>();
+
+    public ChannelUpdater() {
+        setName("DiscordSRV - Channel Updater");
+    }
+
+    public void reload() {
+        // Deleting and recreating the list of updater channels
+        this.updaterChannels.clear();
+
+        final List<Map<String, Object>> configEntries = DiscordSRV.config().getList("ChannelUpdater");
+        for (final Map<String, Object> configEntry : configEntries) {
+            final String channelId = configEntry.get("ChannelId").toString();
+            final String format = configEntry.get("Format").toString();
+            final String intervalAsString = configEntry.get("UpdateInterval").toString();
+            final int interval;
+
+            if (channelId.equals("0000000000000000")) continue; // Ignore default
+
+            if (StringUtils.isAnyBlank(channelId, format)) {
+                DiscordSRV.error("Failed to initialise a ChannelUpdater entry: Missing either ChannelId or Format");
+                continue;
+            }
+            if (StringUtils.isNotBlank(intervalAsString) && StringUtils.isNumeric(intervalAsString)) {
+                interval = Integer.parseInt(intervalAsString);
+            } else {
+                DiscordSRV.debug(Debug.CHANNEL_UPDATER, "Update interval provided for Updater Channel " + channelId + " was blank or invalid, using the default value of 10");
+                interval = 10;
+            }
+
+            final GuildChannel channel = DiscordUtil.getJda().getGuildChannelById(channelId);
+            if (channel == null) {
+                DiscordSRV.error("ChannelUpdater entry " + channelId + " has an invalid id");
+                continue;
+            }
+            DiscordSRV.debug(Debug.CHANNEL_UPDATER, "Initialising ChannelUpdater entry " + channelId);
+            this.updaterChannels.add(new UpdaterChannel(channel, format, interval));
+        }
+    }
+
+    @Override
+    public void run() {
+        reload();
```

Comment:
Reloading on every loop would reset the timer on every updater

Reply:
What do you mean by this? It's only reloaded at the start and with /discord reload.

Reply:
Seems like a looked at that wrong before